src/components/code/index.tsx

```diff
@@ -62,77 +224,19 @@ const Code: FC<CodeProps> = ({
     }, 800);
   };
 
-  const download = () => {
-    if (downloadCallback) {
-      downloadCallback();
-    } else {
-      saveToFile(title || 'download', content);
-    }
-  };
-
   return (
-    <Flex width="100%" sx={{ ...styles, p: padding }} height="100%">
-      <Flex width="100%" sx={{ ...codeHeaderStyles }}>
-        {title ? (
-          <Flex alignItems="center" ml="8px">
-            {title}
-          </Flex>
-        ) : (
-          <div />
-        )}
-        <Flex>
-          {downloadButton && (
-            <Box>
-              <Button
-                intent="ghost"
-                sx={{ ...buttonsStyles }}
-                onClick={download}
-              >
-                {icons.download}
-                <Value ml="5px" mt="1px">
-                  download
-                </Value>
-              </Button>
-            </Box>
-          )}
-          {copyButton && (
-            <Box>
-              <Button
-                intent="ghost"
-                sx={{ ...buttonsStyles }}
-                onClick={useCodeKey}
-                disabled={copied}
-              >
-                {icons.copy}
-                <Value ml="5px" mt="1px">
-                  {!copied ? 'copy' : 'copied'}
-                </Value>
-              </Button>
-            </Box>
-          )}
-        </Flex>
-      </Flex>
-      <Flex width="100%" variant="code" height="100%" {...props}>
-        {element || (
-          <SyntaxHighlighter
-            wrapLongLines={wrapLongLines}
-            showLineNumbers={showLineNumbers}
-            lineNumberStyle={{
-              ...lineNumberStyles,
-              background: theme.colors.grayShade1,
-            }}
-            language={isColorSyntax ? language : 'text'}
-            customStyle={{
-              ...boxStyles,
-              paddingLeft: showLineNumbers ? '0px' : '20px',
-            }}
-          >
-            {content}
-          </SyntaxHighlighter>
-        )}
-      </Flex>
-    </Flex>
+    <Box>
+      <Button
+        intent="ghost"
+        sx={{ ...buttonsStyles }}
```

Comment:
why spread here? Can it be `sx={buttonStyles}`?

Reply:
good point, old stuff but gonna fix this ;)